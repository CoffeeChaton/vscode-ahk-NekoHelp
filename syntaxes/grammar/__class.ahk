Class ±µ·¼，ӿ9 {
    
}