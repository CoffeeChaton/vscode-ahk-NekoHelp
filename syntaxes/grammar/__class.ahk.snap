>﻿Class ±µ·¼，ӿ9 {
#^^^^^^^^^^^^^^^ source.ahk
#               ^ source.ahk punctuation.bracket.ahk
>
>}
#^ source.ahk punctuation.bracket.ahk
>
>Class ±µ·¼，ӿ8 {
#^^^^^ source.ahk meta.block.class.ahk storage.type.class.ahk
#     ^ source.ahk meta.block.class.ahk
#      ^^^^^^^ source.ahk meta.block.class.ahk entity.name.type.class.ahk
#             ^^ source.ahk meta.block.class.ahk
>    __new() {
#^^^^ source.ahk
#    ^^^^^ source.ahk entity.name.function.ahk
#         ^ source.ahk punctuation.bracket.ahk
#          ^ source.ahk punctuation.bracket.ahk
#           ^ source.ahk
#            ^ source.ahk punctuation.bracket.ahk
>
>    }
#^^^^ source.ahk
#    ^ source.ahk punctuation.bracket.ahk
>}
#^ source.ahk punctuation.bracket.ahk
>
>obj8 := new ±µ·¼，ӿ8()
#^^^^^ source.ahk
#     ^^ source.ahk keyword.operator.assignment.ahk
#       ^ source.ahk
#        ^^^^ source.ahk keyword.other.operator.hasdoc.ahk
#            ^^^^^^^ source.ahk entity.name.type.class.ahk
#                   ^ source.ahk punctuation.bracket.ahk
#                    ^ source.ahk punctuation.bracket.ahk
>obj9 := new ±µ·¼，ӿ9
#^^^^^ source.ahk
#     ^^ source.ahk keyword.operator.assignment.ahk
#       ^ source.ahk
#        ^^^^ source.ahk keyword.other.operator.hasdoc.ahk
#            ^^^^^^^ source.ahk entity.name.type.class.ahk
>